lib/ansible/module_utils/facts/virtual/linux.py

```diff
@@ -152,6 +152,12 @@ def get_virtual_facts(self):
                 virtual_facts['virtualization_type'] = 'RHEV'
                 found_virt = True
 
+        if product_name == 'RHEL':
```

Comment:
This is wrong, it will require further discussion within RHV team

Reply:
yes, because "RHEL" also means regular RHEL VMs (e.g. virt-manager created), not RHV

Reply:
Let's remove this whole condition, above code should be enough